README.md

```diff
@@ -358,7 +358,7 @@ to
 
 Next we'll use the `default_barista_base_url` in the code to avoid hard-coding the location of the `barista` service.
 
-Edit the file `open-liberty-masterclass/start/coffee-shop/src/main/java/com/sebastian_daschner/coffee_shop/control/Barista.java`
+Edit the file `open-liberty-masterclass/start/coffee-shop/src/main/java/com/sebastian_daschner/coffee_shop/control/HealthResource.java`
 
```

Comment:
Is this not already the correct link? `HealthResource.java` doesn't exist under `coffee_shop/control/`, its under `coffee_shop/boundary/`. I think this comes from the fact that the file is called a different name between start/finish, under start it's `HealthResouce.java` and under finish it's called `CoffeeShopHealth.java`.